### DISCUSSION TITLE
[2321310] [2313240] [2551170] Hidden Cats In Town & Cats Hidden in Italy & Cats Hidden in Maple Hollow 🍂

### DISCUSSION THREAD

**Discussion by ZEERDEER** [pjy612/SteamManifestCache#51] [Share]
2313240看没有公开密钥 所以也分享了，要找到3个库里面没有的游戏太麻烦了，而且基本都有了
;-(
[config.vdf& Manifest.zip](https://github.com/pjy612/SteamManifestCache/files/13691202/config.vdf.Manifest.zip)

**Comment by pjy612** [comment]
感谢丰富库存~
